wifi.lua

```diff
@@ -3,7 +3,46 @@ minetest.register_node("more_chests:wifi", {
 	tiles = {"wifi_top.png", "wifi_top.png", "wifi_side.png",
 		"wifi_side.png", "wifi_side.png", "wifi_front.png"},
 	paramtype2 = "facedir",
-	groups = {snappy=2, choppy=2, oddly_breakable_by_hand=2,},
+	groups = {snappy=2, choppy=2, oddly_breakable_by_hand=2, tubedevice = 1, tubedevice_receiver = 1},
+	-- Pipeworks
+	tube = {
+		insert_object = function(pos, node, stack, direction, owner)
+			if not owner then
+				return stack
+			end
+			local player = minetest.get_player_by_name(owner)
+			if not player then
+				return stack
+			end
+			local inv = player:get_inventory()
+			return inv:add_item("more_chests:wifi", stack)
+		end,
+		can_insert = function(pos, node, stack, direction, owner)
+			if not owner then
+				return false
+			end
+			local player = minetest.get_player_by_name(owner)
+			if not player then
+				return false
+			end
+			local inv = player:get_inventory()
+			return inv:room_for_item("more_chests:wifi", stack)
+		end,
+		input_inventory = "more_chests:wifi",
```

Comment:
Does that work with detached inventories? :astonished:

Reply:
If I remember correctly, this here is the listname.
The inventory is gotten via `return_input_invref` (here's the pipeworks commit that made this possible: https://gitlab.com/VanessaE/pipeworks/-/commit/a5e1bfa0e824ab70877803d087c97440ff729cac#38deb38da9132073cae6df77fe1bf77a55727e02_322_323).
Also note that the wifi chest uses player inventories, not detached.